### PR TITLE
Display model creation errors

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #1558.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- deploy the GUI with Juju
- deploy a model using the deployment flow
- try and deploy a second model using the same name as the first one
- you should get an error notification

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5930/
Test PASSed.

File: jujugui/static/gui/src/test/test_utils.js [hunk 1]
```diff
@@ -1109,6 +1114,19 @@ describe('utilities', function() {
...
       assert.equal(callback.callCount, 1);
     });
+
+    it('can display an error notification', function() {
+      var model = {
+        name: 'koala',
+        uuid: 'uuid123'
+      };
+      utils._newModelCallback(
+        app, callback, 'Error: no Tasmanian Tigers were found', model);
```
Comment (frankban) [inline jujugui/static/gui/src/test/test_utils.js]: That's a shame, at least one tasmanian tiger should always be there...

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

File: jujugui/static/gui/src/test/test_utils.js (hunk 1 above)
Comment (huwshimi) [inline jujugui/static/gui/src/test/test_utils.js]: It truly is a shame.

**Pull request merged**